.github/workflows/validate-build-e2e.yml

```diff
@@ -0,0 +1,61 @@
+name: Validate Build (E2E tests)
+
+on:
+  push:
+    branches:
+      - main
+      - dajusto/run-tests-in-gh-action
+    paths-ignore: [ '**.md' ]
+  pull_request:
+    branches:
+      - main
+    paths-ignore: [ '**.md' ]
+
+env:
+  solution: WebJobs.Extensions.DurableTask.sln
+  config: Release
+  AzureWebJobsStorage: UseDevelopmentStorage=true
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+
+    - name: Set up .NET Core 3.1
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '3.1.x'
+
+    - name: Set up .NET Core 2.1
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '2.1.x'
+
+    - name: Restore dependencies
+      run: dotnet restore $solution
+
+    - name: Build
+      run: dotnet build $solution
+
+    # Install Azurite
+    - name: Set up Node.js (needed for Azurite)
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18.x' # Azurite requires at least Node 18
+
+    - name: Install Azurite
+      run: npm install -g azurite
+
+    # Run tests
+    - name: Run FunctionsV2 tests (only DurableEntity_CleanEntityStorage test, which is flaky)
+      run: azurite --silent --blobPort 10000 --queuePort 10001 --tablePort 10002 & dotnet test ./test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj  --filter "FullyQualifiedName~Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests.DurableTaskEndToEndTests.DurableEntity_CleanEntityStorage"
```

Comment:
What's the benefit of running this test separately because it's flaky? Is it just to separate it so the report is more explicit and we don't have to look up if this test failed?

Reply:
My mental model is that flakyness is caused by resource contention, so I run it separately from all other tests to the VM has plenty of resources to finish the test in time / so that Azurite isn't overwhelmed

Reply:
Ok, thanks for the explanation!